internal/pkg/hub/manifest.go

```diff
@@ -7,36 +7,12 @@ import (
 	"net/http"
 	"net/http/httputil"
 
+	"github.com/confluentinc/cli/internal/pkg/ccstructs"
 	"github.com/confluentinc/cli/internal/pkg/errors"
 	"github.com/confluentinc/cli/internal/pkg/log"
 )
 
-type Manifest struct {
```

Comment:
I got an import cycle error since this package calls `testserver` for the test URL, but I wanted to use this struct in the hub handler file. Couldn't put this struct in the `connect` package for the same reason (that package imports this one).

So I dumped this struct into our internal `ccstructs` package.